packages/wrangler/src/index.tsx

```diff
@@ -936,6 +936,30 @@ export async function main(argv: string[]): Promise<void> {
       const accountId = !args.local ? await requireAuth(config) : undefined;
 
       // TODO: if worker_dev = false and no routes, then error (only for dev)
+      /**
+       * Collects routes based on priorityto be aggregated,
+       * if no routes are found returns`undefined`
+       */
+      function collectPriorityRoutes() {
+        if (typeof args.routes !== "undefined") {
+          return args.routes;
+        }
+        if (typeof config.routes !== "undefined") {
+          const configRoutes = [];
+          for (const route of config.routes) {
+            if (typeof route === "object") {
+              configRoutes.push(route.pattern);
+            }
+            if (typeof route === "string") {
+              configRoutes.push(route);
+            }
+          }
+          return configRoutes;
+        }
+        if (typeof config.route === "object") {
```

Comment:
what if `config.route` is a string?

Reply:
Missed that check, I will add it in. Thanks for the catch.